### PR TITLE
README.md links to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MaxWofford**
This addresses hackedu/meta#131

### CODE REVIEW

**Comment by jonleung** [comment]
You have no idea how good it felt to see this pull request @MaxWofford! THANKS!

**Comment by MaxWofford** [comment]
@jonleung This is based on spacemacs' [README.md](https://github.com/syl20bnr/spacemacs/blob/master/README.md#contributions)

File: README.md [hunk 1]
```diff
@@ -28,3 +28,9 @@ Build hackEDU with us. Here are two ways to get started:
 
 - [Create an issue](https://github.com/hackedu/hack-camp/issues)
 - [Join us on IRC](https://webchat.freenode.net/?channels=hackedu)
+
+## Contributing
+
+If you are ready to contribute please begin by consulting the
```
Comment (jonleung) [inline README.md]: Actually @MaxWofford, thoughts on changing `If you are ready to contribute` to `If you would like to contribute`?

**Comment by MaxWofford** [comment]
@jonleung Sure, I'll add another commit ontop of this branch. Would be awesome if these could come up before merge in the future.

**Comment by jonleung** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<a name="top"></a>
+ <a name="top"></a>
```
Comment (zachlatta) [inline README.md]: Is there a reason this extra space was added?

File: README.md [hunk 3]
```diff
@@ -28,3 +28,9 @@ Build hackEDU with us. Here are two ways to get started:
 
 - [Create an issue](https://github.com/hackedu/hack-camp/issues)
 - [Join us on IRC](https://webchat.freenode.net/?channels=hackedu)
+
+## Contributing
+
+If you are ready to contribute please begin by consulting the
+[contribution guidelines](CONTRIBUTING.md).
+
```
Comment (zachlatta) [inline README.md]: Is there a reason there's an extra newline here?

**Pull request merged**